artemis/src/test/java/tech/pegasys/artemis/state/BeaconStateTest.java

```diff
@@ -51,23 +51,27 @@ private BeaconState newState() {
     // Initialize state
```

Comment:
Is it possible to incorporate get_initial_beacon_state() into this so that there is less code to maintain?

Reply:
For testing simplicity sake, it's better to hard-code IMO.